packages/amplify-category-api/src/provider-utils/awscloudformation/awsConstants.ts

```diff
@@ -0,0 +1,7 @@
+import path from 'path';
```

Comment:
I think we have a constants file in src/ - can't we re-use that?

Reply:
I made this file because these constants are specific to the aws cfn provider so they don't really apply across the whole category (although there's only one provider)